pandas/io/excel/_base.py

```diff
@@ -165,31 +165,12 @@
     Supported engines: "xlrd", "openpyxl", "odf", "pyxlsb", "calamine".
```

Comment:
Out of scope, but it appears to me this line is duplicative.

Reply:
I removed this line